docs/BLUE-GREEN-DEPLOYMENT.md

```
@@ -0,0 +1,298 @@
+│   ├── nginx.active.conf              # Symlink to active configuration
+│   └── snippets/
+│       └── ssl-params.conf            # Shared SSL parameters
+└── scripts/
+    └── switch-environment.sh          # Environment switching script
+```
+
+## Usage
+
+### Switching Environments
+
+```bash
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)